src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -0,0 +1,212 @@
+        protected override IDeviceActionService deviceActionService => _deviceActionService;
+        protected override IPlatformUtilsService platformUtilsService => _platformUtilsService;
+
+        public void StopCheckLoginRequestStatus()
+        {
+            try
+            {
+                _checkLoginRequestStatusCts?.Cancel();
+                _checkLoginRequestStatusCts?.Dispose();
+            }
+            catch (Exception ex)
+            {
+                _logger.Exception(ex);
+            }
+        }
+
+        public void StartCheckLoginRequestStatus()
+        {
+            try
+            {
+                _checkLoginRequestStatusCts?.Cancel();
+                _checkLoginRequestStatusCts = new CancellationTokenSource();
+                _checkLoginRequestStatusTask = new TimerTask(_logger, CheckLoginRequestStatus, _checkLoginRequestStatusCts).RunPeriodic(TimeSpan.FromSeconds(REQUEST_TIME_UPDATE_PERIOD_IN_SECONDS));
+            }
+            catch (Exception ex)
+            {
+                _logger.Exception(ex);
+            }
+        }
+
+        private async Task CheckLoginRequestStatus()
+        {
+            if (string.IsNullOrEmpty(_requestId) || string.IsNullOrEmpty(_requestAccessCode))
+            {
+                return;
+            }
+
+            try
+            {
+                var response = await _authService.GetPasswordlessLoginResponse(_requestId, _requestAccessCode);
+
+                if (!response.RequestApproved)
+                {
+                    return;
+                }
+
+                StopCheckLoginRequestStatus();
+
+                var decKey = await _cryptoFunctionService.RsaDecryptAsync(response.Key, _requestKeyPair.Item2);
+                var decPasswordHash = await _cryptoFunctionService.RsaDecryptAsync(response.MasterPasswordHash, _requestKeyPair.Item2);
+
+                var authResult = await _authService.LogInPasswordlessAsync(Email, _requestAccessCode, _requestId, decKey, decPasswordHash);
+                await AppHelpers.ResetInvalidUnlockAttemptsAsync();
+
+                if (authResult.CaptchaNeeded)
+                {
+                    if (await HandleCaptchaAsync(authResult.CaptchaSiteKey))
+                    {
+                        await CheckLoginRequestStatus();
+                        _captchaToken = null;
+                    }
+                    return;
+                }
+                _captchaToken = null;
+
+                await _deviceActionService.HideLoadingAsync();
+
+                if (authResult.TwoFactor)
+                {
+                    StartTwoFactorAction?.Invoke();
+                }
+                else if (authResult.ForcePasswordReset)
+                {
+                    UpdateTempPasswordAction?.Invoke();
+                }
+                else
+                {
+                    var task = Task.Run(async () => await _syncService.FullSyncAsync(true));
```

Comment:
There's no need for setting the task to a local var. Also you don't have any exception handling there. This could be done simply by:
```c#
_syncService.FullSyncAsync(true).FireAndForget();
```